src/cli.mjs

```diff
@@ -36,13 +34,14 @@ const __dirname = path.dirname(__filename);
 
 /**
  * @param {string|string[]} _argvInput - The command string provided by the user or test. Parsed by yargs into an argv object.
- * @param {cliContainer} _container - A built and ready for use awilix container with registered injectables.
+ * @param {import('./config/container.mjs').container} _container - A built and ready for use awilix container with registered injectables.
  */
 export async function run(_argvInput, _container) {
-  container = _container;
+  setContainer(_container);
+
   const argvInput = _argvInput;
-  const logger = container.resolve("logger");
-  const parseYargs = container.resolve("parseYargs");
+  const logger = _container.resolve("logger");
```

Comment:
rename to `container`? the declaration these used to conflict with no longer exists